### PR TITLE
update git cookbook with my aliases

### PR DESCRIPTION
**Pull request by ryanmeador**
Add info on three aliases I commonly use: short status, better log, and submodule update.

### CODE REVIEW

File: git_cookbook.md [hunk 1]
```diff
@@ -20,3 +20,23 @@ function pr () {
...
+Includes only the files that changed and omits the boilerplate about the branch, etc.  Very handy to alias as `git s`.
+```
+git status -s
+``` 
+
+#### A better overview of history
+This command lists commits one-per-line, in color, with a graph of the history on the left.  Very handy for quickly looking at history or checking the relationships between branches.  I alias it as `git lg`.
+
+```
+git log --oneline --decorate --graph --all -100
```
Comment (Benuuu) [inline git_cookbook.md:52]: Don't think I would include `-100` in this cookbook.

**Comment by Benuuu** [comment]
:ship:

**Comment by ryanmeador** [comment]
I can see making a note of what it does, and I'd be happy to do that.  Why not include it at all?  It can get really slow with infinite history, and I've found 100 commits to be a sweet spot for almost every purpose.

File: git_cookbook.md (hunk 1 above)
Comment (Benuuu) [inline git_cookbook.md:52]: Making a note is fine too.  The description suggests that it is a better view of history, which implies all history.  This is why I wouldn't include it.  Does your history viewer not handle long `--oneline` history well?  I think `less` has handled it fine for me.

Comment (ryanmeador) [inline git_cookbook.md:52]: I use `less`.  The issue isn't the viewing, it's the generating.  Even if you don't look at all of it, it has to assemble it all, and `less` won't show you the contents until the stream closes (AFAICT).  I'll add a note.

**Comment by gdaniels** [comment]
Can we remove the first-person "I" references?

**Comment by ryanmeador** [comment]
Sure.  Do you think I should include my name, or try to rephrase to not mention anyone at all?

**Comment by brightredchilli** [comment]
I like the 'Very handy to alias as X' format of saying it.

**Comment by gdaniels** [comment]
Keeping it imperative is generally good IMO.  "You might want to run this after every..." or "Aliasing this as `git lg` is handy" etc.

**Comment by ryanmeador** [comment]
I changed the wording.  Good to go?

**Comment by brightredchilli** [comment]
:ship:

**Comment by gdaniels** [comment]
:shipit: :two:

**Pull request merged**